src/plugins/piral-breadcrumbs/src/useBreadcrumbs.ts

```diff
@@ -0,0 +1,54 @@
+import { useGlobalState } from 'piral-core';
+import { BreadcrumbRegistration } from './types';
+
+function getExact(breadcrumbs: Array<BreadcrumbRegistration>, path: string) {
+  const [bc] = breadcrumbs.filter((m) => m.matcher.test(path));
```

Comment:
[Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) would do the job.


What happens if there are multiple registrations, that match the given path. Wouldn't the registration order dictate the outcome of `getExact`?

Reply:
No - https://caniuse.com/array-find (always have an eye for IE11; we should avoid polyfills if they don't bring any advantage).

We only take the first one.